docs/source/getting-started/index.rst

```diff
@@ -77,9 +54,61 @@ While the specifics of each tool differ, most tools will pick up your account in
 
    * - Environment variable
      - Value
+   * - ``MYTHX_API_KEY``
+     - JWT Token
    * - ``MYTHX_ETH_ADDRESS``
      - Your MythX account (Ethereum address)
    * - ``MYTHX_PASSWORD``
-     - API key supplied to you during registration
+     - Password set during registration
+     
+Authentication
+--------------
+
+Once your account is set up, head over to the `dashboard <https://dashboard.mythx.io/>`_.
+In the *Profile* section, various means of authentication are presented.
+
+
+Using API Tokens
+~~~~~~~~~~~~~~~~
+
+This is the recommended way of authenticating with the MythX smart contract
+analysis API. In the *Profile* section there is an elements labeled "MythX API Key".
```

Comment:
Not a big deal, but it stands out better when using the `guilabel` inline element when discussing stuff on the screen.